pages/instrumenting.md

```diff
@@ -125,12 +125,28 @@ information for the request.
 * Flags - Provides the ability to create and communicate feature flags. This is how
 we can tell downstream services that this is a "debug" request.
 
-Check [here](https://github.com/openzipkin/brave/blob/e474ed1e1cd291c7ebc6830c58fdba0a6318fdd2/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java) for the format
+Check [here](https://github.com/openzipkin/brave/blob/e474ed1e1cd291c7ebc6830c58fdba0a6318fdd2/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java) for the format.
 
 Finagle provides mechanisms for passing this information with HTTP and Thrift
 requests. Other protocols will need to be augmented with the information for
 tracing to be effective.
 
+**Sampling decisions are made at the edge of the system**
+
+Downstream services must honour the sampling decision of the upstream system.
+If there's no "Sampled" information in the incoming request, the library should
+make a decision on whether to sample this request, and include the decision in
+further downstream. requests. This provides several desirable properties:
+
+ * Traces always start at the most-upstream service with Zipkin integration.
+ * Since the sampling decision is made at the edge of the system, we'll always
+   have consistent traces - either the whole call graph is traced, or none of
+   it.
+ * Cumulative sample rates from sampling decisions in downstream services is
+   eliminated, Sample rate is clearly controlled at the edge of the system supporting
+   Zipkin, ensuring accurate control of the sample rate (happens at the edge of
+   the system).
+
```

Comment:
Some caveots: debug flag will force the trace to be sampled. This applies even to storage tier. Ex if storage tier enables sampling, debug traces will still be retained